test/test_builders.py

```diff
@@ -237,6 +242,75 @@ def compare_confs(entity_path, conf, cells):
         assert yaml.dump(clean_cells(cells)) == yaml.dump(expected_cells)
 
 
+@pytest.fixture
+def mockSegImagePyramidBuilder():
```

Comment:
We should update this to match naming conventions, looks good otherwise!

```suggestion
def mock_seg_image_pyramid_builder():
```